src/helpers/utils.ts

```diff
@@ -90,3 +90,15 @@ export function getIp(req) {
 
   return ips[0].trim();
 }
+
+export function verifyAuth(req, res, next) {
+  const auth = req.headers.secret || '';
+  const authHash = sha256(auth);
+  const secretHash = process.env.AUTH_SECRET;
```

Comment:
Can we rename `AUTH_SECRET`  so we remember it is related to laser or do we plan to use it for more things in the future?